package.json

```diff
@@ -34,7 +34,10 @@
     "react-helmet": "^5.2.0",
     "react-icons": "^3.5.0",
     "react-lines-ellipsis": ">0.14.0",
+    "react-markdown": "^4.3.1",
     "react-responsive-embed": "^2.1.0",
+    "react-truncate-markdown": "^0.0.1",
```

Comment:
If we're not using these 2 packages, we should remove them from the package.json file.